src/components/Gesture.svelte

```diff
@@ -235,14 +274,32 @@
             <i class="w-full h-full m-0 mt-4 p-2 fas fa-check fa-2x text-secondary" />
           </div>
           <StandardButton
-            onClick={recordClicked}
+            onClick={() => {
+              showCountdown = true;
+              countdownStart()}}
             small
             shadows={false}
             outlined
             fillOnHover>{$t('content.data.record')}</StandardButton>
         </div>
       {/if}
     </GestureTilePart>
+
+    {#if showCountdown === true}
+    <BaseDialog
+      background="light"
+      isOpen={showCountdown}
+      onClose={() => showCountdown = false}>
+      <div class="space-y-10 w-max">
+        <GestureTilePart elevated={true}>
+          <p class="text-9xl text-center">{countdownValue}</p>
+          <p>START ACTION BEFORE THE COUNTDOWN FINISHES</p>
```

Comment:
In the design, this sentence is not in all caps. Also, this string should go in the translations file rather than be hardcoded here.